subprojects/kotlin-dsl/src/integTest/kotlin/org/gradle/kotlin/dsl/support/EmbeddedKotlinProviderTest.kt

```diff
@@ -44,24 +44,22 @@ class EmbeddedKotlinProviderTest : AbstractKotlinIntegrationTest() {
     fun `stdlib and reflect are pinned to the embedded kotlin version for requested plugins`() {
         withBuildScript(
             """
+            buildscript {
+                $repositoriesBlock
+                dependencies {
+                    classpath("org.jetbrains.kotlin:kotlin-stdlib:1.6.10")
```

Comment:
Why are these extra dependencies necessary now? The only thing you need here is the plugin repositories, which could go to `Settings.pluginManagement`, right?

Reply:
This was added because with plugin 1.6.10 `kotlin-stdlib` and `kotlin-reflect` are not seen on `classpath` anymore. I will double check with `kotlin-dsl` experts if that is a good fix.

Reply:
@bamboo do you maybe know if `stdlib` and `reflect` are removed from that classpath on our side or there is some other reason why they are not on `classpath` after upgrade to 1.6.10?

Reply:
The new `kotlin("jvm")` plugin no longer depends kotlin-stdlib and kotlin-reflect. The change to the test looks good 👍